test/norms_test.jl

```diff
@@ -7,15 +7,15 @@
     @test Norm(x, x) ≈ 0.0
     @test Norm(x, y) ≈ abs(4im)
 
-    WeightedNorm = HC.WeightedNorm(Norm, x)
-    @test HC.weights(WeightedNorm) == ones(3)
-    WeightedNorm .= 2.0
-    @test HC.weights(WeightedNorm) == [2.0, 2, 2]
-    WeightedNorm[1] = 4.0
-    @test WeightedNorm isa HC.AbstractNorm
-    @test WeightedNorm(x) ≈ 0.5 * abs(5 + 2.0im)
-    @test WeightedNorm(x, x) ≈ 0.0
-    @test WeightedNorm(x, y) ≈ 0.25 * abs(4im)
+    weighted_norm = HC.WeightedNorm(Norm, x)
+    @test HC.weights(weighted_norm) == ones(3)
+    weighted_norm .= 2.0
```

Comment:
I don't think this will change the failure (unless I'm missing something else).  I think you need to explicitly do:

```suggestion
    weighted_norm.weights .= 2.0
```